requirements.txt

```diff
@@ -1,47 +1,47 @@
-bitarray==2.7.3 ; python_version >= "3.7" and python_version < "4"
```

Comment:
I'd prefer not to include package upgrades in this pr and do them separately. Any reason the existing versions didn't work for you?